tutoraspects/plugin.py

```diff
@@ -38,6 +38,7 @@
         ("DOCKER_IMAGE_RALPH", "fundocker/ralph:3.8.0"),
         ("DOCKER_IMAGE_SUPERSET", "edunext/aspects-superset:{{ ASPECTS_VERSION }}"),
         ("DOCKER_IMAGE_VECTOR", "timberio/vector:0.30.0-alpine"),
+        ("DOCKER_IMAGE_OPENEDX", "edunext/openedx-aspects:{{ ASPECTS_VERSION }}"),
```

Comment:
Will this not try to overwrite the image for all of tutor?

Reply:
No, it will be pushed to the `docker.io/edunext/openedx-aspects` repository. This is to verify if there are any performance benefits for this CI